### DISCUSSION TITLE
Remove iOS?

### DISCUSSION THREAD

**Discussion by NaluriMurai** [YSCEDC/YSCE#3] [General]
Wanted to ask for people's thoughts on the matter since It's something of a.. well, appendix on the source code, given that a probable extreme minority of end users run YSFlight on mobile, let alone iOS - but should we just go ahead and remove the iOS code from the repo?

**Comment by decaff42** [comment]
I don't think it is worth developing. Given what Soji said about paying to host in the app store and all the other things, it isn't really going to be a huge market. You'd have to simplify things a lot and getting addons into the game would be a pain to say the least.

**Reply by NaluriMurai** [reply]
Agreed. I'll make a fork to compile sometime next week when I have free time and see if there's any adverse effects just to be safe.
